### ISSUE TITLE
 Backend can not be started, exit_code=1

### ISSUE THREAD

**Issue by o---** [SpectrumIM/spectrum2#277]
Hi,

getting the following errors repeatedly in the /var/log/spectrum2/whatsapp.example.com/spectrum2.log

2018-03-09 01:42:04,435 INFO  NetworkPluginServer: Tried to spawn first backend with pid 1953
2018-03-09 01:42:04,435 INFO  NetworkPluginServer: Backend should now connect to Spectrum2 instance. Spectrum2 won't accept any connection before backend connects
2018-03-09 01:42:05,435 ERROR NetworkPluginServer: Backend can not be started, exit_code=1, possible error: Operation not permitted
2018-03-09 01:42:05,435 ERROR NetworkPluginServer: Check backend log for more details
2018-03-09 01:42:05,435 INFO  NetworkPluginServer: Starting new backend /usr/bin/transwhat.py --host localhost --port 39779 --service.backend_id=1 "-j" "whatsapp.example.com" "/etc/spectrum2/tr
ansports/whatsapp.cfg"

What is it hat's not permitted and how to resolve, please? Thanks.

**Comment by vitalyster** [comment]
It is not permitted to exec transwhat.py because of exec permissions missing?

**Comment by o---** [comment]
Thanks. 

I have changed the transwhat.py file owner to spectrum:spectrum now. However before I did that, without any change, the log changed and now its saying that the file does not exist. But the file does exist and the path is correct. 

2018-02-18 22:04:11,347 INFO  NetworkPluginServer: Tried to spawn first backend with pid 10107
2018-02-18 22:04:11,347 INFO  NetworkPluginServer: Backend should now connect to Spectrum2 instance. Spectrum2 won't accept any connection before backend connects
2018-02-18 22:11:27,658 INFO  XMPPFrontend: Creating component in gateway mode
2018-02-18 22:11:27,659 INFO  SQLite3Backend: Opening database /var/lib/spectrum2/whatsapp.example.com/database.sql
2018-02-18 22:11:27,660 INFO  NetworkPluginServer: Listening on host localhost port 38693
2018-02-18 22:11:27,660 INFO  NetworkPluginServer: Starting new backend /usr/bin/transwhat.py --host localhost --port 38693 --service.backend_id=1 "/etc/spectrum2/transports/whatsapp.cfg".
2018-02-18 22:11:27,661 INFO  NetworkPluginServer: Tried to spawn first backend with pid 11291
2018-02-18 22:11:27,661 INFO  NetworkPluginServer: Backend should now connect to Spectrum2 instance. Spectrum2 won't accept any connection before backend connects
2018-02-18 22:11:28,661 ERROR NetworkPluginServer: Backend can not be started, exit_code=2, possible error: No such file or directory
2018-02-18 22:11:28,661 ERROR NetworkPluginServer: This usually means the path to backend executable defined in config file as '[service] backend="..."' is wrong or the executable does not exists.
2018-02-18 22:11:28,661 ERROR NetworkPluginServer: Check backend log for more details
2018-02-18 22:11:28,661 INFO  NetworkPluginServer: Starting new backend /usr/bin/transwhat.py --host localhost --port 38693 --service.backend_id=1 "/etc/spectrum2/transports/whatsapp.cfg".
2018-02-18 22:11:28,662 INFO  NetworkPluginServer: Tried to spawn first backend with pid 11298
